test/helpers/kube_dump.go

```diff
@@ -364,7 +376,7 @@ func ControllerDumpOnFail(ctx context.Context, kubectlCli *kubectl.Cli, outLog i
 					portforward.WithPorts(int(admin.AdminPort), int(admin.AdminPort)),
 				)
 				if err != nil {
-					fmt.Printf("error starting port forward to controller admin port: %f\n", err)
+					fmt.Printf("error starting port forward to controller admin port: %s\n", err.Error())
```

Comment:
Thank you, my bad.

Reply:
I'm going to blame that one on GitHub Copilot. I wonder if there is a linter that checks for silly things like treating an `error` as a float/number.

Reply:
Probably. I recall someone adding this type of thing in GME, or maybe it was just for error wrapping.

Reply:
I looked and didn't see a linter that would catch this case. I may make one in my personal time, after I catchup on bug reports and feature requests with my current neglected linter (sqlclosecheck).